packages/motion/src/helpers.js

```diff
@@ -85,7 +85,7 @@ export function getWebpackConfig(state: State, config: Motion$Config, cli: CLI,
         Path.join(Path.normalize(Path.join(__dirname, '..')), 'node_modules'),
         Path.join(Path.normalize(Path.join(__dirname, '..', '..')), 'node_modules')
       ],
-      packageMains: ['webpack', 'browser', 'web', 'browserify', 'jsnext:main', 'main']
+      packageMains: ['webpack', 'browser', 'web', 'browserify', 'main', 'jsnext:main']
```

Comment:
Yep, though we can see how it should be handled and do the transpile I think. For now I just put it at the end so it doesn't cause a bug.